### PR TITLE
PXC-4154: [MTR] Integrate galera-x mtr suite

### PR DESCRIPTION
**Pull request by voodam**
Reason for deleting tests:

* `x_random_password` says: "Percona XtraDB Cluster doesn't allow use of RANDOM PASSWORD for CREATE/ALTER USER operation while operating in cluster mode", so it seems it useless here.
* `notices_gr_*` tests: I thought that group replication is not combined with group replication normally. Anyway, that tests are just copy-pasted from the `x` suite.

### CODE REVIEW

**Comment by voodam** [comment]
https://pxc.cd.percona.com/job/pxc-8.0-pipeline-parallel-mtr/1615/

**Comment by voodam** [comment]
https://pxc.cd.percona.com/job/pxc-8.0-pipeline-parallel-mtr/1627/

**Comment by venkatesh-prasad-v** [comment]
@voodam Hi! Can you please rebase your branch on 8.0 branch and run these tests?

As I see, the following tests are failing on 8.0
```
galera-x.mysqlxtest_help galera-x.crud_insert_generated_ids galera-x.admin_create_index_spatial galera-x.insert_table_bad_column_type galera-x.insert_crud_o galera-x.performance_schema_stmt galera-x.create_index_crud
```
and some tests may need re-recording. Please check this.

**Comment by voodam** [comment]
> @voodam Hi! Can you please rebase your branch on 8.0 branch and run these tests?
> 
> As I see, the following tests are failing on 8.0
> 
> ```
> galera-x.mysqlxtest_help galera-x.crud_insert_generated_ids galera-x.admin_create_index_spatial galera-x.insert_table_bad_column_type galera-x.insert_crud_o galera-x.performance_schema_stmt galera-x.create_index_crud
> ```
> 
> and some tests may need re-recording. Please check this.

Done

**Comment by voodam** [comment]
https://pxc.cd.percona.com/job/pxc-8.0-pipeline-parallel-mtr/1634/

File: mysql-test/suite/galera-x/t/connection_skip_grant_table.test [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## This is test file for connection authentication using mysqlx.
+##
+--source include/have_util_sed.inc
+--source include/galera_cluster.inc
+--sleep 60
```
Comment (venkatesh-prasad-v) [inline mysql-test/suite/galera-x/t/connection_skip_grant_table.test:5]: Why do we have this sleep here?

**Review by venkatesh-prasad-v — COMMENTED** [review]
Please squash your commits before merging the PR.

File: mysql-test/suite/galera-x/t/connection_skip_grant_table.test (hunk 1 above)
Comment (voodam) [reply mysql-test/suite/galera-x/t/connection_skip_grant_table.test:5]: Just copied from the codership release. Anyway, this test is not fixed and disabled for now.

**Review by venkatesh-prasad-v — APPROVED** [review]

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**